src/main/java/org/openrewrite/java/spring/boot2/upgrade/to25/AnnotateClassesDependingOnDataSource.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache License 2.0
+ *
+ * @author: fkrueger
+ */
+package org.openrewrite.java.spring.boot2.upgrade.to25;
```

Comment:
for consistency let's move this to the `org.openrewrite.java.spring.boot2` package.  Also, it can be added to the [spring-boot 2.5](https://github.com/openrewrite/rewrite-spring/blob/b50312cab5f59044d42b3a7eaa4a9c0934ea04f6/src/main/resources/META-INF/rewrite/spring-boot2-upgrade-version.yml#L216)  upgrade recipes.

Reply:
Hm... imagining we want to provide recipes for all 40+ "steps" just for 2.5, I thought another package for separation might be a good idea?

Reply:
thanks :)

Reply:
Moved all classes p to boot2 package as reqested.